flash/core/model.py

```diff
@@ -123,6 +127,8 @@ def predict(
         skip_collate_fn: bool = False,
         dataloader_idx: Optional[int] = None,
         data_pipeline: Optional[DataPipeline] = None,
+        return_type: str = None,
+        labels: Optional[List[str]] = None,
```

Comment:
is it updated also in docs?

Reply:
@Borda are docs created automatically with help of docstrings?

Reply:
yes, types and default are also in docs